<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgWelcomeRecentProjectEntry</class>
 <widget class="QWidget" name="WdgWelcomeRecentProjectEntry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>82</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0,0">
   <item>
    <widget class="QLabel" name="lIcon">
     <property name="minimumSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="text">
      <string>&lt;Icon&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="lName">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&lt;Name&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lPath">
       <property name="text">
        <string>&lt;Path&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lLastUsed">
       <property name="text">
        <string>&lt;LastUsed&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pbOpen">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>42</width>
       <height>42</height>
      </size>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
     <property name="icon">
      <iconset resource="../Resources.qrc">
       <normaloff>:/Icons/classic/folder-open.ico</normaloff>:/Icons/classic/folder-open.ico</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pbRemove">
     <property name="minimumSize">
      <size>
       <width>42</width>
       <height>42</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Remove from list</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../Resources.qrc">
       <normaloff>:/Icons/classic/circle-xmark.ico</normaloff>:/Icons/classic/circle-xmark.ico</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Resources.qrc"/>
 </resources>
 <connections/>
</ui>
